src/main/java/core/basesyntax/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
```

Comment:
This class should be in the `service.impl` package.

Reply:
changes have been made